### DISCUSSION TITLE
Concentric top and bottom layer

### DISCUSSION THREAD

**Discussion by FlaMtnBkr** [OrcaSlicer/OrcaSlicer#4852] [Ideas]
Is there a way to make a concentric pattern so it's similar to the vase mode where there is a continuous line instead of starting and stopping for each line which also creates a seam? 

I've been using concentric patterns more recently on the top and bottom which seems to give me a better surface. But I think it could be even better if it was more of a constant spiral. It would really work well for round shapes but even square could benefit and really I think any shape would do well. It would just have to start with a bit of a tapered line or even overlap the wall a little to achieve the taper and then each line would move inward at this spot instead of stopping, moving, and starting.

Just throwing the idea out there to see if anyone else might think it could be beneficial? 

Thanks!

**Comment by foresterr** [comment]
Have you tried the "Archimedean chords" top/bottom pattern? On circular areas it's pretty much what you describe; in more complex shapes it gets tricky.
